### PR TITLE
User Sign Up/Profile Flow

### PR DESCRIPTION
**Pull request by emjayholla**
- Create account is functioning. 
- User profile fields are now editable and will save in the database. 
- SQL files have been changed, *be sure to drop the user_profile table and re-run it*

### CODE REVIEW

**Review by LaplacesAngel — COMMENTED** [review]
For some reason can't get the submit button to work after hitting signup:
image


Also not able to load the profile page:
image


Not sure if it's just me or not, maybe someone else could take a look? 
Or Maggie are there potentially changes on your local branch that haven't been pushed up yet if it's working for you?

**Comment by Dalton-Kujawa** [comment]
Hey Maggie I could create a user through sign up but is the profile page support to work at all? its crashing the back end due to some table joining issues.

**Review by Dalton-Kujawa — CHANGES_REQUESTED** [review]
Hey Maggie, things are working well and look good on both pages. I have to request changes only because both sign up and profile need some changes before we can commit them into main. They both need error handling on screen. As it is right now if a user puts in a field that isnt allowed (aka. A phone number thats already used, or a putting in Texas instead of TX, or using too many characters for some names) it will crash the backend. I think just a little bit of verification on the front end input should fix that right up! Also in .vscode/settings.json you have eslint.enable set to false. This would get rid of all those nice red squiggles that tell use when we are making mistakes in our code. ( although they can be annoying)

File: backend/data/db.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (Dalton-Kujawa) [inline backend/data/db.json:1]: I have no idea where this file came from lol this is from my original push weeks ago!

**Review by Dalton-Kujawa — APPROVED** [review]
BackendErrorCrash
I am going to approve the code but I did get a backend crash for some of the inputs on profile. If you can make sure you pull from main before you Merge the branch in and make sure you dont have any conflicts! Thanks!

**Pull request merged**